Loggie/Classes/Utility/Data+FormattedJsonString.swift

```diff
@@ -14,7 +14,8 @@ extension Data {
         guard let jsonObject = try? JSONSerialization.jsonObject(with: self, options: .allowFragments) else {
             return nil
         }
-        guard let formattedBody = try? JSONSerialization.data(withJSONObject: jsonObject, options: .prettyPrinted) else {
+        guard JSONSerialization.isValidJSONObject(jsonObject),
```

Comment:
Hmm shouldn't this check be in guard before, since if child object is not valid then the whole object is not valid, or am I missing something?